jetstream/experimenter.py

```diff
@@ -139,13 +139,22 @@ class ExperimentCollection:
 
     @classmethod
     def from_experimenter(
-        cls, session: requests.Session = None, with_draft_experiments=False
+        cls,
+        session: requests.Session = None,
+        with_draft_experiments=False,
+        slug=None,
     ) -> "ExperimentCollection":
         session = session or requests.Session()
 
-        nimbus_experiments_json = retry_get(
-            session, cls.EXPERIMENTER_API_URL_V8, cls.MAX_RETRIES, cls.USER_AGENT
-        )
+        url = cls.EXPERIMENTER_API_URL_V8
+        draft_url = cls.EXPERIMENTER_API_URL_V8_DRAFTS
+        if slug:
+            url = url + slug + "/"
+            draft_url = draft_url + slug + "/"
```

Comment:
Probably want to use [urljoin](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin) here?

Reply:
Fine with me, but do we gain anything from using `urljoin` here? Also, we'll still need to tack on the `+ "/"` at the end since the API needs that and `urljoin` won't add it.